<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Subscriber</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_sub"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="Add">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Delete">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resign">
          <property name="text">
           <string>resign</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="updateS">
        <property name="text">
         <string>Update subscriber list</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="list_sub">
        <property name="cursor" stdset="0">
         <cursorShape>OpenHandCursor</cursorShape>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>FileManager</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_fm"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="DeleteFile">
          <property name="text">
           <string>DeleteFile</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="AddFile">
          <property name="text">
           <string>AddFile</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="updateF">
        <property name="text">
         <string>Update files list</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="list_fm"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Add</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addSubscriber()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Delete</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteSubscriber()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DeleteFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>694</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resign</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>resign()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateS</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateSubscribers()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateF</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>596</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addSubscriber()</slot>
  <slot>deleteSubscriber()</slot>
  <slot>deleteFile()</slot>
  <slot>addFile()</slot>
  <slot>resign()</slot>
  <slot>updateSubscribers()</slot>
  <slot>updateFiles()</slot>
 </slots>
</ui>
